docs/docs/en/contributors/releases.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Releases
+titleTemplate: :title | Contributors | Tuist
+description: Learn how Tuist's continuous release process works
+---
+
+# Releases
+
+Tuist uses a continuous release system that automatically publishes new versions whenever meaningful changes are merged to the main branch. This approach ensures that improvements reach users quickly without manual intervention from maintainers.
+
+## Overview
+
+We continuously release three main components:
+- **Tuist CLI** - The command-line tool
+- **Tuist Server** - The backend services
+- **Tuist App** - The macOS desktop application (see [iOS App Exception](#ios-app-exception))
```

Comment:
the iOS app is still continuously deployed to TestFlight, but yeah, the App Store releases are manually. 
```suggestion
- **Tuist App** - The macOS and iOS apps (iOS app is only continuously deployed to TestFlight, see more [here](#app-store-release)
```

I'd also rename the section to either `App Store release` or `iOS app release`. `iOS app exception` title sounds a bit weird, to me, anyways.